cmd/tempo/app/modules.go

```diff
@@ -248,13 +249,23 @@ func (t *App) initStore() (services.Service, error) {
 }
 
 func (t *App) initMemberlistKV() (services.Service, error) {
-	t.cfg.MemberlistKV.MetricsRegisterer = prometheus.DefaultRegisterer
+	reg := prometheus.DefaultRegisterer
+	t.cfg.MemberlistKV.MetricsRegisterer = reg
 	t.cfg.MemberlistKV.MetricsNamespace = metricsNamespace
 	t.cfg.MemberlistKV.Codecs = []codec.Codec{
 		ring.GetCodec(),
 	}
 
-	t.MemberlistKV = memberlist.NewKVInitService(&t.cfg.MemberlistKV, log.Logger)
+	dnsProviderReg := prometheus.WrapRegistererWithPrefix(
+		"cortex_",
```

Comment:
probably don't want to use the `cortex_` provider here. `tempo_`?

Reply:
Good catch. I'll update that.